staging/src/k8s.io/sample-apiserver/pkg/client/clientset_generated/clientset/typed/wardle/v1alpha1/fischerlist.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	watch "k8s.io/apimachinery/pkg/watch"
+	rest "k8s.io/client-go/rest"
+	v1alpha1 "k8s.io/sample-apiserver/pkg/apis/wardle/v1alpha1"
+	scheme "k8s.io/sample-apiserver/pkg/client/clientset_generated/clientset/scheme"
+)
+
+// FischerListsGetter has a method to return a FischerListInterface.
+// A group's client should implement this interface.
+type FischerListsGetter interface {
+	FischerLists() FischerListInterface
+}
+
+// FischerListInterface has methods to work with FischerList resources.
+type FischerListInterface interface {
+	Create(*v1alpha1.FischerList) (*v1alpha1.FischerList, error)
+	Update(*v1alpha1.FischerList) (*v1alpha1.FischerList, error)
+	Delete(name string, options *v1.DeleteOptions) error
+	DeleteCollection(options *v1.DeleteOptions, listOptions v1.ListOptions) error
+	Get(name string, options v1.GetOptions) (*v1alpha1.FischerList, error)
+	List(opts v1.ListOptions) (*v1alpha1.FischerListList, error)
```

Comment:
Only the generator uses the annotation. And as we throw away the clientset before this PR, everything is fine. Moreover, we don't use this clientset (yet) anywhere, so go build does not complain.